src/autopas/containers/verletListsCellBased/verletListsCells/traversals/VLCCellPairC08CellHandler.h

```diff
@@ -115,13 +107,17 @@ class VLCCellPairC08CellHandler : public LCC08CellHandler<ParticleCell, Pairwise
             const auto &currentList = soaNeighborList[offsetCell2][cell2LocalNoN3->second];
             for (const auto &[particleIndex, particleList] : currentList) {
               if (not particleList.empty()) {
-                pairwiseFunctor->SoAFunctorVerlet(*soa, particleIndex, particleList, this->_useNewton3);
+                pairwiseFunctor->SoAFunctorVerlet(*soa, particleIndex, particleList, useNewton3);
               }
             }
           }
         }
       }
     }
   }
+
+ private:
+  /** Member containng the cell pair offsets for processCellListsC08 */
+  std::vector<internal::OffsetPair> _cellPairOffsets;
```

Comment:
```suggestion
  /** 
   * Member containng the cell pair offsets for processCellListsC08 
   */
  std::vector<internal::OffsetPair> _cellPairOffsets;
```

I don't really care but for consistency sake I'd use the same style as everywhere else in AutoPas